### PR TITLE
Create new aggregate table fx_health_ind_searches_by_provider_v1

### PR DESCRIPTION
**Pull request by kwindau**
## Description

This PR creates the following new table: 
- moz-fx-data-shared-prod.telemetry_derived.fx_health_ind_searches_by_provider_v1

## Related Tickets & Documents
* [DENG-7021](https://mozilla-hub.atlassian.net/browse/DENG-7021)

**Reviewer, please follow [this checklist](https://github.com/mozilla/bigquery-etl/blob/main/.github/reviewer_checklist.md)**

[DENG-7021]: https://mozilla-hub.atlassian.net/browse/DENG-7021?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/DENG-7078)

### CODE REVIEW

File: sql/moz-fx-data-shared-prod/telemetry_derived/fx_health_ind_searches_by_provider_v1/metadata.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+friendly_name: Fx Health Ind Searches By Provider
```
Comment (scholtzan) [inline sql/moz-fx-data-shared-prod/telemetry_derived/fx_health_ind_searches_by_provider_v1/metadata.yaml:1]: What does the "ind" stand for?

Comment (kwindau) [reply sql/moz-fx-data-shared-prod/telemetry_derived/fx_health_ind_searches_by_provider_v1/metadata.yaml:1]: Good catch, I'll fix that in the metadata to put the full name - basically these are a set of aggregate tables all used in a dashboard called the "Firefox Health Indicator" Dashboard, so I was prefixing all of them the same to keep them together and putting them all in the same DAG

File: sql/moz-fx-data-shared-prod/telemetry_derived/fx_health_ind_searches_by_provider_v1/metadata.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+friendly_name: Firefox Health Indicator Searches By Provider
+description: |-
+  Count of searches & users by provider/day used in Firefox Health Indicator dashboard
+owners:
+- kwindau@mozilla.com
+labels:
+  incremental: true
+  owner1: kwindau@mozilla.com
+  table_type: aggregate
```
Comment (kik-kik) [inline sql/moz-fx-data-shared-prod/telemetry_derived/fx_health_ind_searches_by_provider_v1/metadata.yaml:9]: Should this also include the `shredder_mitigation` label?

File: sql/moz-fx-data-shared-prod/telemetry_derived/fx_health_ind_searches_by_provider_v1/query.sql [hunk 3]
```diff
@@ -0,0 +1,13 @@
+SELECT
+  submission_date_s3 AS submission_date,
+  default_search_engine,
+  SUM(search_count_all) AS searches,
+  COUNT(DISTINCT(client_id)) AS users
```
Comment (kik-kik) [inline sql/moz-fx-data-shared-prod/telemetry_derived/fx_health_ind_searches_by_provider_v1/query.sql:5]: Do we want any additional dimensions here? For example to be able to break this down by country or anything like that?

File: sql/moz-fx-data-shared-prod/telemetry_derived/fx_health_ind_searches_by_provider_v1/metadata.yaml (hunk 2 above)
Comment (kwindau) [reply sql/moz-fx-data-shared-prod/telemetry_derived/fx_health_ind_searches_by_provider_v1/metadata.yaml:9]: Good catch, added! thanks!

File: sql/moz-fx-data-shared-prod/telemetry_derived/fx_health_ind_searches_by_provider_v1/schema.yaml [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+  type: STRING
+  description: Default Search Engine
+- mode: NULLABLE
+  name: searches
+  type: INTEGER
+  description: Number of Searches
+- mode: NULLABLE
+  name: users
+  type: INTEGER
+  description: Number of Users
```
Comment (kik-kik) [inline sql/moz-fx-data-shared-prod/telemetry_derived/fx_health_ind_searches_by_provider_v1/schema.yaml:17]:
```suggestion
  description: Number of distinct users.
```

**Review by kik-kik — APPROVED** [review]
rw+

File: sql/moz-fx-data-shared-prod/telemetry_derived/fx_health_ind_searches_by_provider_v1/query.sql (hunk 3 above)
Comment (kwindau) [reply sql/moz-fx-data-shared-prod/telemetry_derived/fx_health_ind_searches_by_provider_v1/query.sql:5]: For this dashboard, this is all the dimensions we need

**Pull request merged**